src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -96,12 +96,12 @@ class SidebarScreen extends Component {
                                 {
                                     icon: ChatBubble,
                                     text: 'New Chat',
-                                    onSelected: () => Navigation.navigate(ROUTES.NEW_CHAT),
+                                    onSelected: () => Navigation.navigate(ROUTES.IOU_REQUEST),
                                 },
                                 {
                                     icon: Users,
                                     text: 'New Group',
-                                    onSelected: () => Navigation.navigate(ROUTES.NEW_GROUP),
+                                    onSelected: () => Navigation.navigate(ROUTES.IOU_BILL),
```

Comment:
Please reverse this, this is just for testing :)

Reply:
Done :)